lib/hanami/helpers/html_helper/empty_html_node.rb

```diff
@@ -124,7 +124,11 @@ def boolean_attribute(attribute_name, _value)
         end
 
         def attribute(attribute_name, value)
-          %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
+          if attribute_name.to_sym == :value
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{Hanami::Utils::Escape.html(value)}")
+          else
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
```

Comment:
We have some attributes like a `script`. If we escape this attributes we will have a error in layout. What do you think what we should do with this attributes?